app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ViewProfileScreenTest.kt

```diff
@@ -314,6 +324,7 @@ class ViewProfileScreenTest {
     composeTestRule.waitForIdle()
 
     composeTestRule.onNodeWithTag("NoFavoritesMessage").assertIsDisplayed()
+    Thread.sleep(5000)
```

Comment:
maybe this could be removed so the CI doesn't take too long :) ?

Reply:
of course